_bacon-gravityforms.scss

```
@@ -216,24 +216,34 @@ label.hidden_sub_label,
   font-size: 0;
 
   > span {
-    display: inline-block;
-    vertical-align: top;
+    display: block;
     font-size: 1rem;
 
-    &.ginput_full {
-      width: 100%;
+    @media only screen and (max-width: 640px) {
+      & + span {
+        margin-top: $gfield-spacing-y * 1.5;
+      }
     }
 
-    &.ginput_right,
-    &.ginput_left,
-    &.name_first,
-    &.name_last {
-      width: calc(50% - (#{$gfield-spacing-x} / 2));
+    @media only screen and (min-width: 641px) {
+      display: inline-block;
+      vertical-align: top;
+
+      &.ginput_right,
+      &.ginput_left,
+      &.name_first,
+      &.name_last {
+        width: calc(50% - (#{$gfield-spacing-x} / 2));
```

```suggestion
        width: calc(50% - #{$gfield-spacing-x * 0.5});
```

Would this work? I'm guessing the output would just be something like `calc(50% - 10px);` instead of `calc(50% - (20px / 2));`

---

The `#{$var}` syntax means that the value gets printed inline, not calculated by Sass. Meaning that we could use custom properties, for example, as the variable value and the browser would work it out using `calc()`. With your suggestion, a non-math value would cause a compilation error, I think.

---

@colourgarden Made a quick codepen: https://codepen.io/wrux/pen/bGejNQG

Here's the output:
```css
.test1 {
    width: calc(50% - (20px / 2));
}

.test2 {
    width: calc(50% - 10px);
}
```